chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -321,32 +331,35 @@ Zotero.QuickCopy = new function() {
 				}
 				
 				var content = {
-					text: contentType == "html" ? html : text,
+					text: format.contentType == "html" ? html : text,
 					html: copyHTML
 				};
 				
 				return content;
 			}
 			
+			// determine locale preference
+			var locale = format.locale ? format.locale : Zotero.Prefs.get('export.quickCopy.locale');
+			
```

Comment:
The validation being that the locale matches the locale defined in the style (if that's the case). A mismatch might not actually cause any problems though, so maybe don't worry about it.